build/serve.js

```diff
@@ -136,6 +137,7 @@ gulp.task('spawn-backend:prod', ['build-frontend', 'backend', 'kill-backend'], f
       [
```

Comment:
Can you move all three arguments of the binary to a common constant in the top of the file? The list of arguments is growing - let's avoid desyncing it.